blueocean-dashboard/src/main/js/components/testing/TestResults.jsx

```diff
@@ -91,78 +92,50 @@ export default class TestResult extends Component {
         const newFailures = tests.filter(t => (t.age <= 1 && t.status === 'FAILED') || t.status === 'REGRESSION');
         const existingFailures = tests.filter(t => t.age > 1 && t.status === 'FAILED');
 
-        let passBlock = null;
         let newFailureBlock = null;
         let existingFailureBlock = null;
         let fixedBlock = null;
         let skippedBlock = null;
-        let summaryBlock = null;
-
-        if (testResults.failCount === 0) {
-            passBlock = (
-                <EmptyStateView iconName="done_all">
-                    <h1 style={{ marginTop: '2.4rem' }}>{translation('rundetail.tests.results.passing.all', { defaultValue: 'All tests are passing' })}</h1>
-                    <p>{translation('rundetail.tests.results.passing.count', {
-                        0: testResults.passCount,
-                        defaultValue: 'Nice one! All {0} tests for this pipeline are passing.',
-                    })}</p>
-                </EmptyStateView>
-            );
-        } else {
-            summaryBlock = (
-                <div className="test-summary">
-                    <div className={`new-passed count-${fixed.length}`}>
-                        <div className="count">{fixed.length}</div>
-                        <label>{translation('rundetail.tests.results.fixed', { defaultValue: 'Fixed' })}</label>
-                    </div>
-                    <div className={`new-failed count-${newFailures.length}`}>
-                        <div className="count">{newFailures.length}</div>
-                        <label>{translation('rundetail.tests.results.failures.new', { defaultValue: 'New' })}</label>
-                    </div>
-                    <div className={`failed count-${testResults.failCount}`}>
-                        <div className="count">{testResults.failCount}</div>
-                        <label>{translation('rundetail.tests.results.failures', { defaultValue: 'Failures' })}</label>
-                    </div>
-                    <div className={`passed count-${testResults.passCount}`}>
-                        <div className="count">{testResults.passCount}</div>
-                        <label>{translation('rundetail.tests.results.passing', { defaultValue: 'Passing' })}</label>
-                    </div>
-                    <div className={`skipped count-${testResults.skipCount}`}>
-                        <div className="count">{testResults.skipCount}</div>
-                        <label>{translation('rundetail.tests.results.skipped', { defaultValue: 'Skipped' })}</label>
-                    </div>
-                </div>
-            );
```

Comment:
`summaryBlock` is replaced with a new component (see below). The other changes below to failures, skipped, etc are just whitespace.